desci-contracts/scripts/deployLocalSubgraph.sh

```diff
@@ -23,21 +23,6 @@ TARGET_ADDRESS=$(jq -r '.proxies[-1].address' "$FILE" || echo "")
 # File doesn't exist when running outside of docker, empty is fine then
 FOUND_ADDRESS=$(grep 0x subgraph/subgraph.yaml | awk '{print $2}' | tr -d '"' || echo "[NONE]")
 
-waitForNodeAdminServer() {
-    until [ \
-        "$(curl -s -w '%{http_code}' -o /dev/null "http://host.docker.internal:8020")" \
-        -eq 405 ]; 
-    do
-        echo "[waitForNodeAdminServer]: Waiting for http://host.docker.internal:8020"
-        sleep 5
-    done
-
-    # until $(curl --output /dev/null --silent --fail http://host.docker.internal:8020); do
-    #     echo "[deployLocalSubgraph] waiting for http://host.docker.internal:8020 to start"
-    #     sleep 5
-    # done
-}
-
```

Comment:
I'm not able to reproduce the earlier problems with the health checks in place, and this snippet broke things on my end with some timing issue. Can give it another go if someone still faces trouble with this :) 

Making a proper healthcheck on the `graph-node` container is very tricky though since it has basically nothing installed.